### PR TITLE
Feat/add platforms and logging

### PR DESCRIPTION
**Pull request by molnosikristof**

### CODE REVIEW

File: e2e/utils/visual-regression.js [hunk 1]
```diff
@@ -3,9 +3,30 @@ function getMisMatchPercentage(results) {
 }
 
 module.exports.takeScreenshotAndGetWholePageCompareResult = (misMatchTolerance) => {
-  return ((getMisMatchPercentage(browser.checkViewport()) <= misMatchTolerance) || false);
+  const misMatchPercentage = getMisMatchPercentage(browser.checkViewport());
+  const isTestPassed = (misMatchPercentage <= misMatchTolerance) || false;
+  if (isTestPassed) {
+    return isTestPassed;
+  }
+  console.log('misMatchTolerance: ' + misMatchTolerance
```
Comment (oroce) [inline e2e/utils/visual-regression.js]: this is really verbose imho. usually we dont need this. my favourite tool for logging verbose messages for myself but not for others is [`debug`](https://www.npmjs.com/package/debug).

This is one the standard ways of logging this in node community

Comment (molnosikristof) [reply e2e/utils/visual-regression.js]: Thanks for the advice,what about I add the debug in an other PR?

File: e2e/utils/visual-regression.js [hunk 2]
```diff
@@ -3,9 +3,30 @@ function getMisMatchPercentage(results) {
...
+  let ignoreComparisonValue = '';
+  if (ignoreComparison === true) {
+    ignoreComparisonValue = 'colors';
+  }
+  const misMatchPercentage = getMisMatchPercentage(browser.checkElement(elementSelector, { ignoreComparison: ignoreComparisonValue }));
+  const isTestPassed = (misMatchPercentage <= misMatchTolerance) || false;
+  if (isTestPassed) {
+    return isTestPassed;
+  }
+  console.log('misMatchTolerance: ' + misMatchTolerance
```
Comment (kristof0425) [inline e2e/utils/visual-regression.js]: Your solution here is totally fine. 👍 

I'm just letting you know that you can ease your life a bit with the so-called [template-literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals):

You can write this `console.log()` without concatenation like this and have the same output:

```js
console.log(`mismatchTolearance: ${misMatchTolerance}
misMatchPercentage: ${misMatchPercentage}
problematic elementSelector: ${elementSelector}`);
```

File: e2e/utils/visual-regression.js (hunk 1 above)
Comment (oroce) [reply e2e/utils/visual-regression.js]: sure!

**Pull request merged**